routing.rst

```diff
@@ -161,7 +161,8 @@ Use the ``methods`` option to restrict the verbs each route should respond to:
              */
             public function show(int $id)
             {
-                // ... return a JSON response with the post
+                // create your json response
+                return $this->json('Hello Symfony');
```

Comment:
This is not consistent with the route definition, I would keep it as is instead.